sys/usb/usbus/usbus_control.c

```diff
@@ -213,21 +212,19 @@ static int _recv_dev_setup(usbus_t *usbus, usb_setup_t *pkt)
             case USB_SETUP_REQ_SET_ADDRESS:
                 DEBUG("usbus_control: Setting address\n");
                 usbus->addr = (uint8_t)pkt->value;
-                res = 0;
+                res = 1;
```

Comment:
Any reason to change this ? Usually we consider 0 to be OK and < 0 not OK

Reply:
This must follow the same return codes from the `control_handler` function:
 - Negative on error, results in USB stall
 - Zero when it is not handled, results in an USB nack if nothing handles the request
 - Positive for the length, which is useful for USB IN transactions, but doesn't really mean anything for OUT transactions where the host decides the amount of data.

This last case could be documented a bit better for OUT transactions

Reply:
It is used to determine the response to the transfer here:
https://github.com/RIOT-OS/RIOT/blob/c4b103306c81ce84eb3bceab80508839a6900172/sys/usb/usbus/usbus_control.c#L277-L294